conans/client/generators.py

```diff
@@ -51,14 +52,14 @@ def content(self):
                         'SET(CONAN_EXE_LINKER_FLAGS_{dep} "{deps.exelinkflags}")\n'
                         'SET(CONAN_C_FLAGS_{dep} "{deps.cflags}")\n')
 
-        for dep_name, dep_cpp_info in self._build_info.dependencies:
+        for dep_name, dep_cpp_info in self._deps_build_info.dependencies:
```

Comment:
Very good rename, the build_info used was actually _deps_build_info. Now all generators have available both, with the proper name, well seen, thanks!